src/export/InstanceExporter.ts

```diff
@@ -314,6 +314,17 @@ export class InstanceExporter implements Fishable {
     // Once all rules are set, we should ensure that we did not add a duplicate profile URL anywhere
     if (instanceDef.meta?.profile) instanceDef.meta.profile = uniq(instanceDef.meta.profile);
 
+    // If we added a versioned profile that's the same as the InstanceOf profile, remove the InstanceOf profile
+    const versionedInstanceOf =
+      instanceDef.meta?.profile?.filter(profile =>
+        profile.startsWith(`${instanceOfStructureDefinition.url}|`)
+      ) ?? [];
+    if (versionedInstanceOf.length > 0) {
+      instanceDef.meta.profile = instanceDef.meta.profile.filter(
+        profile => profile !== instanceOfStructureDefinition.url
+      );
```

Comment:
Maybe this is an edge case, but... what if the author wanted `meta.profile` to list both and explicitly stated so:
```
* meta.profile[+] = "http://myig.org/fhir/StructureDefinition/FooProfile"
* meta.profile[+] = "http://myig.org/fhir/StructureDefinition/FooProfile|4"
```
This would filter out the author-provided URL without a version.  I think, if it is possible, it would be better to just _stop_ the auto-population of `meta.profile` if we detect a versioned (or unversioned) URL for the `InstanceOf` already in there.  (But I'm not sure where that code lives or how feasible it is).

Reply:
This seems reasonable to me. Most authors won't be explicitly adding the unversioned profile, so I think it's safe to guess that any authors who do so definitely want to keep it around.